src/components/utils/paths.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @author psarando
+ *
+ * Utils for building page paths.
+ */
+import NavigationConstants from "../../common/NavigationConstants";
+
+/**
+ * Builds a path to the App Launch Wizard for the app with the given IDs.
+ *
+ * @param {string} systemId The app's system ID.
+ * @param {string} appId The app's ID.
+ */
+export const getAppLaunchPath = (systemId, appId) =>
+    `/${NavigationConstants.APPS}/${systemId}/${appId}/launch`;
```

Comment:
This can be moved to`src/components/apps/utils.js`. We already have `utils.js` for analyses and data :)